fedimint-core/Cargo.toml

```diff
@@ -26,30 +26,30 @@ bls12_381 = "0.7.1"
 serdect = "0.2.0"
 itertools = { workspace = true }
 jsonrpsee-types = { version = "0.22.2" }
-jsonrpsee-core = { version = "0.22.2", features = [ "client" ] }
+jsonrpsee-core = { version = "0.22.2", features = ["client"] }
```

Comment:
unrelated: which one do you like more?
I prefer the top one in toml because it "feels right"

Reply:
I don't care, wasn't this done by our `just format`? Is my IDE suddenly formatting toml files?! If so I'm sorry, I'll teach it some manners.

Reply:
yep, it is your IDE. The changes are good, we need toml formatter in fedimint